### PR TITLE
Fix GCE tests

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
For whatever reason, I overlooked this in #10758. Probably because I made the changes in the dashboard and then assumed I was "done with Kubermatic".

**Does this PR introduce a user-facing change? Then add your Release Note here**:
```release-note
NONE
```

### CODE REVIEW

File: hack/local/run-api-e2e.sh [hunk 1]
```diff
@@ -110,7 +110,7 @@ metadata:
   namespace: kubermatic
 spec:
   gcp:
-    serviceAccount: ${GOOGLE_SERVICE_ACCOUNT}
+    serviceAccount: "${GOOGLE_SERVICE_ACCOUNT}"
```
Comment (xmudrii) [inline hack/local/run-api-e2e.sh:113]: Why are those two not base64 encoded?

Comment (xrstf) [reply hack/local/run-api-e2e.sh:113]: They are, in the chunk above (line 43).

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve

**Pull request merged**